File: app2/src/main/java/com/ssz/framejava/module/splash/SplashActivity.java
package com.ssz.framejava.module.splash;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;

import com.ssz.framejava.R;
import com.ssz.framejava.T.Test;
import com.ssz.framejava.base.BaseSplashActivity;
import com.ssz.framejava.module.home.HomeActivity;


/**
 * @author : zsp
 * time : 2019 11 2019/11/5 15:33
 */
public class SplashActivity extends BaseSplashActivity {

    @Override
    protected void afterOnCreate(@Nullable Bundle savedInstanceState) {

        startActivity(new Intent(this, HomeActivity.class));
        // 不需要转场动画
        overridePendingTransition(0, 0);

        // 触发测试（T）
        new Test().startTest();

        finish();
    }

    @Override
    protected long getDelayTime() {
        return super.getDelayTime();
    }

    @Override
    protected void resetTheme() {
        setTheme(R.style.AppTheme_NoTitle);
    }
}

File: app/src/main/java/com/ssz/studydemo/ui/splash/SplashActivity.kt
package com.ssz.studydemo.ui.splash

import android.content.Intent
import com.ssz.studydemo.R
import com.ssz.studydemo.base.BaseSplashActivity
import com.ssz.studydemo.ui.dispatch.DispatchActivity

/**
 * @author : zsp
 * time : 2019 10 2019/10/24 13:10
 *
 * <style name="AppTheme.NoTitle">
 * <item name="windowNoTitle">true</item>
 * <item name="windowActionBar">false</item>
 * </style>
 *
 * <style name="AppTheme.NoTitle.FullScreen">
 * <item name="android:windowFullscreen">true</item>
 * </style>
 */
class SplashActivity : BaseSplashActivity() {

    override fun resetTheme() {
        setTheme(R.style.AppTheme_NoTitle)
    }

    /**
     * 获取Layout,默认不设置
     */
    override fun getLayoutId(): Int {
        return super.getLayoutId()
    }

    /**
     * 初始化逻辑
     */
    override fun afterOnCreate() {
        startActivity(Intent(this, DispatchActivity::class.java))
        overridePendingTransition(0, 0)
        finish()
//        post(Runnable {
//        },3000)
    }
}